flags/test.ts

```diff
@@ -1,6 +1,10 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 import { assertEquals } from "../testing/asserts.ts";
 import { parse } from "./mod.ts";
+import {
+  assert,
+  IsExact,
+} from "https://deno.land/x/conditional_type_checks@1.0.5/mod.ts";
```

Comment:
We prefer to avoid depending on `deno.land/x/` from `std`. Can you copy the necessary part of the module in std as test utility?

Reply:
Yep i know, it was just temporarly 😉 will do 👍